app/src/test/java/com/teamagam/gimelgimel/app/integration/interactors/DisplayVectorLayerInteractorTest.java

```diff
@@ -131,7 +131,7 @@ public void setFirstVisibleThenExecuteThenSetSecondVisible_BothShouldBeVisible()
   }
 
   private Scheduler createTestScheduler() {
-    return Schedulers.immediate();
+    return Schedulers.trampoline();
```

Comment:
I know, but it's a test so it's not supposed to influence the app's performance.
I understand the cons of trampoline but it's the best solution exists in rx2 as for my understanding.